feature/settings/src/androidTest/java/com/google/jetpackcamera/settings/CameraAppSettingsViewModelTest.kt

```diff
@@ -79,7 +79,7 @@ internal class CameraAppSettingsViewModelTest {
 
     @Test
     fun getSettingsUiState() = runTest(StandardTestDispatcher()) {
-        // giving viewmodel time to call init, otherwise settings will stay disabled
+        // giving view model time to call init, otherwise settings will stay disabled
```

Comment:
nit: is this being caught as a typo? Lets call it ViewModel, I think this will fix it.